### PR TITLE
Added example how to zero-pad floating-point number

### PR DESCRIPTION
**Pull request by KES777**

### CODE REVIEW

File: cpan/Pod-Perldoc/corpus/perlfunc.pod [hunk 1]
```diff
@@ -6423,11 +6423,12 @@ how many places right of the decimal point to show (the default being 6).
...
-  printf '<%.1f>', 1;  # prints "<1.0>"
-  printf '<%.0f>', 1;  # prints "<1>"
-  printf '<%e>', 10;   # prints "<1.000000e+01>"
-  printf '<%.1e>', 10; # prints "<1.0e+01>"
+  printf '<%f>', 1;       # prints "<1.000000>"
+  printf '<%.1f>', 1;     # prints "<1.0>"
+  printf '<%.0f>', 1;     # prints "<1>"
+  printf '<%07.2f>', 1.3; # prints "<0001.30>"
+  printf '<%e>', 10;      # prints "<1.000000e+01>"
+  printf '<%.1e>', 10;    # prints "<1.0e+01>"
```
Comment (tonycoz) [inline cpan/Pod-Perldoc/corpus/perlfunc.pod]: This file isn't documentation and isn't maintained by p5p (and doesn't match the core perlfunc.pod)

Comment (KES777) [reply cpan/Pod-Perldoc/corpus/perlfunc.pod]: What can I do here?

Comment (tonycoz) [reply cpan/Pod-Perldoc/corpus/perlfunc.pod]: Don't change it.  It is used as a test file by Pod::Perldoc, not as documentation.

Comment (tonycoz) [reply cpan/Pod-Perldoc/corpus/perlfunc.pod]: Thanks.

Hopefully #21144 will make this less confusing.

**Review by tonycoz — APPROVED** [review]

**Pull request merged**